gcloud/bigtable/row.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""User friendly container for Google Cloud Bigtable Row."""
+
+
+import six
+
+
+def _to_bytes(value, encoding='ascii'):
+    """Converts a string value to bytes, if necessary.
+
+    Unfortunately, ``six.b`` is insufficient for this task since in
+    Python2 it does not modify ``unicode`` objects.
+
+    :type value: str / bytes or unicode
+    :param value: The string/bytes value to be converted.
+
+    :type encoding: str
+    :param encoding: The encoding to use to convert unicode to bytes. Defaults
+                     to "ascii", which will not allow any characters from
+                     ordinals larger than 127. Other useful values are
+                     "latin-1", which which will only allows byte ordinals
+                     (up to 255) and "utf-8", which will encode any unicode
+                     that needs to be.
+
```

Comment:
Yeah, I brought it over from `gcloud_bigtable._helpers`, but didn't want to sprout that module prematurely. It gets used a lot since the distinction between bytes and string(unicode) is important for protobuf fields.

I can just put it in `gcloud._helpers` since it is general purpose. (I even debated using [`oauth2client._helpers._to_bytes`][1] but didn't want to depend on a non-public object in a non-public submodule.

[1]: https://github.com/google/oauth2client/blob/73d9d55447de97dfe541395817a0c8241701f7d6/oauth2client/_helpers.py#L44